src/axolotl/integrations/base.py

```diff
@@ -277,7 +284,7 @@ def get_input_args(self):
         list[str]: A list of Pydantic classes for all registered plugins' input arguments.'
         """
         input_args = []
-        for plugin in self.plugins:
+        for _, plugin in self.plugins.items():
```

Comment:
Just curious, why not `.values()`?

Reply:
Let me do this then, initially when I was writing code I was doing some login with the plugin names for debugging, hence the `.items()` stuck around 😅